Formula/wandio.rb

```diff
@@ -18,6 +19,15 @@ class Wandio < Formula
     sha256 cellar: :any, high_sierra:   "29602aec2851811108e97397e6310f091f5e5fe0844f9cfcd6657193d9c53ff4"
   end
 
+  depends_on "lz4"
+  depends_on "lzo"
+  depends_on "xz" # For LZMA
+  depends_on "zstd"
```

Comment:
At least description claims support for lzma and zstd, but these are not available unless dependencies are available at compile-time (`xz` and `zstd`). Not sure if worth enabling other formats (`lz4` and `lzo`), but added for now.

Existing bottles are probably built with:
```
configure: Compiled with compressed file (zlib) support: Yes
configure: Compiled with compressed file (bz2) support: Yes
configure: Compiled with compressed file (lzo write only) support: No
configure: Compiled with compressed file (lzma) support: No
configure: Compiled with compressed file (zstd) support: No
configure: Compiled with compressed file (lz4) support: No
configure: Compiled with Intel QuickAssist Technology support: No
configure: Compiled with http read (libcurl) support: Yes
```

With dependencies:
```
configure: Compiled with compressed file (zlib) support: Yes
configure: Compiled with compressed file (bz2) support: Yes
configure: Compiled with compressed file (lzo write only) support: Yes
configure: Compiled with compressed file (lzma) support: Yes
configure: Compiled with compressed file (zstd) support: Yes
configure: Compiled with compressed file (lz4) support: Yes
configure: Compiled with Intel QuickAssist Technology support: No
configure: Compiled with http read (libcurl) support: Yes
```

Reply:
Do we want all the extra dependencies?

Reply:
Not sure. Wandio's main feature is dealing with multiple compression formats and their installation recommendation is to add all libraries https://github.com/wanduow/wandio/blob/master/INSTALL
```
Strongly recommended:
 * libpthread
 * zlib-dev
 * libbz2-dev
 * liblzma-dev
 * liblzo2-dev
 * liblz4-dev
 * libzstd-dev
```

I assume they do so for their maintained packages:
- https://github.com/wanduow/wandio/wiki/Installing-via-RPM
- https://github.com/wanduow/wandio/wiki/Installing-via-Debian-Package